code/modules/power/singularity/narsie.dm

```diff
@@ -38,8 +38,8 @@
 	narsie_spawn_animation()
 
 	sleep(70)
-	shuttle_master.emergency.request(null, 0.3) // Cannot recall
-	shuttle_master.emergency.canRecall = FALSE
+	SSshuttle.emergency.request(null, 0.3) // Cannot recall
+	SSshuttle.emergency.canRecall = FALSE
```

Comment:
Just glanced at it. Unblock recall or something like that? Might do that if I decide to work on porting over tg's improvement. For now, I just want to KO everything on the ProcessScheduler -> StonedMC list. It would make future improvement much easier.